test/trento/domain/sap_system/sap_system_test.exs

```diff
@@ -1344,12 +1344,18 @@ defmodule Trento.SapSystemTest do
   describe "deregistration" do
     test "should deregister a Database and SAP system when the Primary database instance is removed" do
       sap_system_id = UUID.uuid4()
-      host_id = UUID.uuid4()
+
+      primary_database_host_id = UUID.uuid4()
       secondary_database_host_id = UUID.uuid4()
+
       deregistered_at = DateTime.utc_now()
+
       db_instance_number_1 = "00"
       db_instance_number_2 = "01"
 
+      db_sid = UUID.uuid4()
```

Comment:
Not asking any change, but using uuid for sids has been always wrong...
The sid is a 3 charts/numbers string...
Not that it matters for tests, but it always raises my ocd detector

Reply:
You're right, my laziness did this, I will write a custom factory for sid so we can have a dedicated tech debt, my goal as you can see was to have the tests close as possibile to the real scenario, I tacked all sids, ids etc, but I agree we need to tackle this right know. Thanks for the suggestion man!

Reply:
I added a custom function to the sap system testing suite for faking sids and add only to the deregistration test, if we like this approach we can merge it, and I'll have a tech deb to wire properly in all the factories